amethyst_input/src/util.rs

```diff
@@ -35,3 +37,15 @@ pub fn is_close_requested(event: &Event) -> bool {
         _ => false,
     }
 }
+
+/// Gets the input axis value from the `InputHandler`.
+/// If the name is None, it will return the default value of the axis (0.0).
+pub fn get_input_axis_simple<A, B>(name: &Option<A>, input: &InputHandler<A, B>) -> f32 
```

Comment:
Why tho?

Reply:
I re-use it heavily in my controllers for hoppinworld and I bet a lot of devs will want to use it too.

Reply:
But why don't you just do `unwrap_or_else`? It's faster to use, simpler and more idiomatic. I don't understand.

Reply:
Ah I forgot to answer this. Because name is an Option. In most controllers, when you declare the system, you can decide to disable an axis by passing None. This function replaces having to do
```rs
let axis_value = if let Some(axis_name) = self.axis1_name {
    input_handler blabla
} else {
  0
};
```
for each of the axis you are checking.